File: app/src/main/java/me/devsaki/hentoid/activities/sources/RedditLaunchActivity.java
package me.devsaki.hentoid.activities.sources;

import android.os.Bundle;

import me.devsaki.hentoid.abstracts.BaseActivity;
import me.devsaki.hentoid.fragments.downloads.RedditLauncherDialogFragment;

/**
 * Created by Robb on 09/2019
 * Landing page history launcher for Reddit
 */
public class RedditLaunchActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        RedditLauncherDialogFragment.invoke(getSupportFragmentManager());
    }
}

File: app/src/main/java/me/devsaki/hentoid/activities/sources/RedditLaunchActivity.kt
package me.devsaki.hentoid.activities.sources

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import me.devsaki.hentoid.fragments.downloads.RedditLauncherDialogFragmentK

/**
 * Landing page history launcher for Reddit
 */
class RedditLaunchActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        RedditLauncherDialogFragmentK.invoke(supportFragmentManager)
    }
}